Cargo.toml

```
@@ -36,7 +36,7 @@ num-format = { version = "0.4.3", default-features = false }
 quick-xml = { version = "0.26", default-features = false }
 rgb = "0.8.13"
 str_stack = "0.1"
-clap = { version = "3.0.1", optional = true, features = ["derive"] }
+clap = { version = "3.2.0", optional = true, features = ["derive"] }
```

```suggestion
clap = { version = "3.2.1", optional = true, features = ["derive"] }
```
Looks like there was a same-day fix release.